src/SAWScript/Interpreter.hs

```diff
@@ -1259,164 +1259,176 @@ primitives = Map.fromList
     , "and the number of nodes it would have if represented as a tree."
     ]
 
-  , prim "assume_valid"        "ProofScript ProofResult"
+  , prim "assume_valid"        "ProofScript ()"
     (pureVal assumeValid)
     Current
-    [ "Assume the current goal is valid, completing the proof." ]
+    [ "Assume the current goal is valid, completing the proof."
+    , "Prefer to use 'admit', this command will eventually be removed."
+    ]
 
-  , prim "assume_unsat"        "ProofScript SatResult"
+  , prim "assume_unsat"        "ProofScript ()"
     (pureVal assumeUnsat)
     Current
-    [ "Assume the current goal is unsatisfiable, completing the proof." ]
+    [ "Assume the current goal is unsatisfiable, completing the proof."
+    , "Prefer to use 'admit', this command will eventually be removed."
+    ]
+
+  , prim "admit"               "String -> ProofScript ()"
```

Comment:
I like this change! Having an explanation for admitting a proof will help generate useful verification summaries, among other benefits.